libraries/AC_WPNav/AC_Circle.cpp

```diff
@@ -142,8 +142,12 @@ void AC_Circle::update()
         // update position controller target
         _pos_control.set_xy_target(target.x, target.y);
 
+        const Vector3f &curr_pos = _inav.get_position();
```

Comment:
Vector2f curr_pos;
        ahrs.get_relative_position_NE_origin(curr_pos);
        curr_pos = curr_pos * 100.0f;  // m to cm